app/views/redirect_to_accept_terms/show.html.erb

```diff
@@ -0,0 +1,6 @@
+<p>If your intention is to file a patent on your idea, use this site cautiously and consult a patent attorney. You should consult a patent attorney regarding any patent related matters including whether submitting a disclosure through FirsttoDisclose.org is appropriate for your situation. The law regarding the effect and extent of public disclosure on your ability to get a U.S. patent is unsettled. Additionally, public disclosure might bar you from obtaining a patent in certain foreign jurisdictions. FirsttoDisclose.org does not warrant or make any claims to help a user obtain a patent.</p>
+<%= form_tag('/redirect_to_accept_terms#update', method: "put") %>
+	<%= label_tag 'Accept'%>
+	<%= check_box_tag 'Accept' %>
+	<%= submit_tag 'Submit' %>
+
```

Comment:
```ruby
<%= form_tag('/accept_terms', method: "put") do %>
    <%= label_tag 'Accept'%>
    <%= check_box_tag 'Accept' %>
    <%= submit_tag 'Submit' %>
<% end %>
```
You need the form_tag do have a do and end to wrap the contents in. You also don't need to specify the controller action in the form_tag, because the put method will automatically try to go to the update action if it exists.